ruby/ql/test/library-tests/dataflow/call-sensitivity/call_sensitivity.rb

```diff
@@ -185,3 +185,21 @@ def method1 x
 end
 
 c = C.new (taint 32)
+
+def invoke_block1 x
+  yield x
+end
+
+def invoke_block2 x
+  invoke_block1 x do |x|
+    yield x
+  end
+end
+
+invoke_block2 (taint 37) do |x|
+  sink x # $ hasValueFlow=37
+end
+
+invoke_block2 "safe" do |x|
+  sink x # $ SPURIOUS hasValueFlow=37
```

Comment:
Is this something we can fix easily, or more tricky?

Reply:
I don't think there is an easy fix, since the data flow library only keeps track of 1 level of call context; in this case we would need 2 levels.